docs/rgb_decomposition.md

```diff
@@ -0,0 +1,53 @@
+RGB Decomposition
+=================
+
+RGB products generated with HyP3 use both co-pol (VV or HH) and cross-pol (VH or HV) backscatter values to generate a color image from SAR data. This approach facilitates visual interpretation by decomposing the signals into surface scattering with some volume scattering (red band), volume scattering (green band), and surface scattering with very low volume scattering (blue band). 
+
+The dual-pol SAR dataset first undergoes radiometric terrain correction (RTC) to adjust for the distortions present in the images due to the side-looking geometry of SAR data acquisition. The RTC images are output in amplitude scale, and the RGB values are calculated using a combination of the VV and VH (or HH and HV) pixel values. The output RGB GeoTIFF is scaled so that the values for each band range from 1 to 255.
+
+### Decomposition Calculations
+
+Note that dual-pol image acquisition may provide either VV/VH returns or HH/HV returns, depending on the sensor mode. The same decomposition approach can be applied to either mode; refer to the appropriate equations below. 
+
+To calculate the RGB values, the data is first separated into surface (P<sub>r</sub>) and volume (P<sub>v</sub>) scattering components as follows:
+
+##### VV/VH
+
+P<sub>r</sub> = |S<sub>vv</sub>| <sup>2</sup> - 3 |S<sub>vh</sub>| <sup>2</sup>
+
+P<sub>v</sub> = 4 |S<sub>vh</sub>| <sup>2</sup>
+
+##### HH/HV
+
+P<sub>r</sub> = |S<sub>hh</sub>| <sup>2</sup> - 3 |S<sub>hv</sub>| <sup>2</sup>
+
+P<sub>v</sub> = 4 |S<sub>hv</sub>| <sup>2</sup>
+
+A spatial mask is generated using a threshold (*k*, typically between -22 and -25 dB) applied to the P<sub>v</sub> data so that pixels with values below the threshold are included in the blue channel mask (M<sub>B</sub>) and those above the threshold are included in the red band mask (M<sub>R</sub>).
+
+M<sub>B</sub> = P<sub>v</sub> < *k*
+
+M<sub>R</sub> = P<sub>v</sub> >= *k*
+
+These spatial masks are then applied to the P<sub>r</sub> data to assign the pixels to either the red or the blue band.
+
+P<sub>B</sub> = P<sub>r</sub> M<sub>B</sub>
+
+P<sub>R</sub> = P<sub>r</sub> M<sub>R</sub>
+
+Finally, the intensity values for each band are multiplied by specific scalars to appropriately stretch the dynamic range of each band to create an RGB:
+
+a<sub>R</sub> = 2 (P<sub>R</sub>) <sup>0.5</sup>
+
+a<sub>G</sub> = 3 / 2 (P<sub>v</sub>) <sup>0.5</sup>
+
+a<sub>B</sub> = 5 (P<sub>B</sub>) <sup>0.5</sup>
+
```

Comment:
There is a Z term that is added in the code to all three of these equations: 

```
z = arctan( 2 (|S<sub>cp</sub>| <sup>2</sup> -  |S<sub>xp</sub>| <sup>2</sup>) <sup>0.5</sup> ) * 2 / 𝜋

a<sub>R</sub> = 2 (P<sub>R</sub>) <sup>0.5</sup> + z

a<sub>G</sub> = 3 / 2 (P<sub>v</sub>) <sup>0.5</sup> + 2 * z

a<sub>B</sub> = 5 (P<sub>B</sub>) <sup>0.5</sup> + 5 * z
```

We also should include how we scale as well, so it'd look like:

```
z = arctan( 2 (|S<sub>cp</sub>| <sup>2</sup> -  |S<sub>xp</sub>| <sup>2</sup>) <sup>0.5</sup> ) * 2 / 𝜋

a<sub>R</sub> = 1 + ( 2 (P<sub>R</sub>) <sup>0.5</sup> + z ) * 254

a<sub>G</sub> = 1 + ( 3 / 2 (P<sub>v</sub>) <sup>0.5</sup> + 2 * z ) * 254

a<sub>B</sub> = 1 + ( 5 (P<sub>B</sub>) <sup>0.5</sup> + 5 * z) * 254
```